src/dailies.ts

```diff
@@ -715,7 +718,7 @@ function internetMemeShop(): void {
 const teas = $items`cuppa Activi tea, cuppa Alacri tea, cuppa Boo tea, cuppa Chari tea, cuppa Craft tea, cuppa Cruel tea, cuppa Dexteri tea, cuppa Feroci tea, cuppa Flamibili tea, cuppa Flexibili tea, cuppa Frost tea, cuppa Gill tea, cuppa Impregnabili tea, cuppa Improprie tea, cuppa Insani tea, cuppa Irritabili tea, cuppa Loyal tea, cuppa Mana tea, cuppa Mediocri tea, cuppa Monstrosi tea, cuppa Morbidi tea, cuppa Nas tea, cuppa Net tea, cuppa Neuroplastici tea, cuppa Obscuri tea, cuppa Physicali tea, cuppa Proprie tea, cuppa Royal tea, cuppa Serendipi tea, cuppa Sobrie tea, cuppa Toast tea, cuppa Twen tea, cuppa Uncertain tea, cuppa Vitali tea, cuppa Voraci tea, cuppa Wit tea, cuppa Yet tea`;
 function pickTea(): void {
   if (!getCampground()["potted tea tree"] || get("_pottedTeaTreeUsed")) return;
-  const bestTea = teas.sort((a, b) => garboValue(b) - garboValue(a))[0];
+  const bestTea = maxBy(teas, garboValue);
   const shakeVal = 3 * garboAverageValue(...teas);
   const teaAction = shakeVal > garboValue(bestTea) ? "shake" : bestTea.name;
```

Comment:
honest thing -- i did not realize garbo had valuation for shaking the tea tree. that's awesome. good job fam.